source/simulator/stokes_matrix_free.cc

```diff
@@ -723,20 +721,26 @@ namespace aspect
 
         for (unsigned int q=0; q<velocity.n_q_points; ++q)
           {
+            VectorizedArray<number> viscosity_x_2;
```

Comment:
Does it make sense to move this variable outside of this loop and initialize with ``(cell, 0)``?  Then you only need the ``else`` here.

Reply:
Yes, that definitely makes sense.